src/servers/src/http.rs

```diff
@@ -708,6 +709,15 @@ impl HttpServer {
                         authorize::check_http_auth,
                     )),
             )
+            .nest(
```

Comment:
> We already have a `/admin` route, why add a new route?

1. `/admin` only exist for metasrv apparently, adding it to all role could be messy?
2. @evenyag suggest add a `debug` route which seems more appropriately: https://github.com/GreptimeTeam/greptimedb/pull/4653#discussion_r1739598842

Reply:
Ok, but it must be under the`/v1/` path, just like `/v1/prof/mem`.

Reply:
> Ok, but it must be under the`/v1/` path, just like `/v1/prof/mem`.

@shuiyisong also suggest reserve `v1` for protocol message only, see: https://github.com/GreptimeTeam/greptimedb/pull/4653#discussion_r1740279857
> We can agree that /{HTTP_API_VERSION} is for user protocol interfaces, but /prof and /debug are not. So let's remove the version prefix. cc

oh and I forget to remove `v1` for `prof/mem`, might also remove memprof port later in separate PR since it's currently broken for some platform(and recommend samply in docs)